js/components/resultsFormPC.js

```diff
@@ -0,0 +1,681 @@
+                    setDisaggregationData={setDisaggregationData}
+                    total={[disaggregationData["653"].labels[0]]}
+                    helptext={helptext}
+                    formErrors={formErrors}
+                />
+                <DissaggregationFields 
+                    indicatorID={indicatorID}
+                    disagg={[disaggregationData["651"]]}
+                    disaggregationData={disaggregationData}
+                    setDisaggregationData={setDisaggregationData}
+                    total={[disaggregationData["653"].labels[1]]}
+                    helptext={helptext}
+                    formErrors={formErrors}
+                />
+
+                <EvidenceFields
+                    evidenceFieldsInput={evidenceFieldsInput}
+                    setEvidenceFieldsInput={setEvidenceFieldsInput}
+                    formErrors={formErrors}
+                />
+
+                <div className="form-actions">
+                    <div>
+                        <button 
+                            type="button" 
+                            className="btn btn-primary result-group" 
+                            id="result-submit-create" 
+                            onClick={(e) => handleSubmit(e)}
+                        >{gettext('Save and close')}
+                        </button>
+                        <button 
+                            type="button" 
+                            className="btn btn-reset result-group" 
+                            id="result-cancel-btn"
+                            data-dismiss="modal"
+                        >{gettext('Cancel')}
+                        </button>
+                    </div>
+                </div>
+            </div>
+        )
+    } else {
+        return (
+            <React.Fragment></React.Fragment> //TODO: Add a waiting spinner/indicator will waiting for API response for data
+        )
+    }
+}
+
+
+
+// ***** Common Fields Section *****
+const CommonFields = ({ commonFieldsInput, setCommonFieldsInput, outcomeThemesData, formErrors }) => {
+
+    useEffect(() => {
+        $('[data-toggle="popover"]').popover({html: true});
+    }, []);
+
+    return (
+        <fieldset>
+            <div className="form-group" id="div_id_date_collected">
+                <label htmlFor="id_date_collected" className="label--required">{gettext('Result date')}</label>
+
+                <a 
+                href="#"
+                tabIndex="0"
+                data-toggle="popover"
+                data-placement="right"
+                data-trigger="focus"
+                data-content={ gettext('If data collection occurred within the fiscal year, enter the date where data was collected. If data collection occurred after the end of the fiscal year, enter the last day of the fiscal year (June 30).')}>
+                    &nbsp;<i className="far fa-question-circle"></i>
+                </a>
```

Comment:
With the exception of a couple of elements, these popover elements are fairly repetitious.  You should be able to build a separate component for these.  There may actually already be one available from the underlying program page.